### ISSUE TITLE
rgis2GPKG admin units

### ISSUE THREAD

**Issue by dvignoles** [bmfekete/RGIS#9]
In the [rgis2GPKG.sh](https://github.com/bmfekete/RGIS/blob/0cac6c2f54fac9ce09d5191065e25a87a11f7134/Scripts/rgis2GPKG.sh) script, geom values are lost for country/state tables in the final step of the `(gdbd|gdbd.gz)` case. 

In  [this function call](https://github.com/bmfekete/RGIS/blob/0cac6c2f54fac9ce09d5191065e25a87a11f7134/Scripts/rgis2GPKG.sh#L162)

`_GPKGattribTable "${SCHEMA}" "${TBLNAME}" "POLYGON" "${ID}" "DN" | spatialite -silent -batch  "${GEOPACKAGE}"`

I've noticed two potential issues: 

1. The value of `$ID` defaults to the string "ID". The correct join column in the case of country/state tables is adm0_code/adm1_code. 
2. In the [SQL block](https://github.com/bmfekete/RGIS/blob/0cac6c2f54fac9ce09d5191065e25a87a11f7134/Scripts/rgis2GPKG.sh#L51) updating the geometry column, multiple geometries may be returned as matched from `${tableName}_geom`. For instance, in the `${tableName}_geom` table, several geometries may share the same `DN` identifier. The union of these geometries makes up the entirety of an administrative unit polygon.

```
echo "SET \"geom\" = (SELECT \"${schemaName}_${tableName}_geom\".\"geom\""
 	echo "                FROM \"${schemaName}_${tableName}_geom\""
	echo "                WHERE \"${schemaName}_${tableName}\".\"${relateID}\" = \"${schemaName}_${tableName}_geom\".\"${joinID}\");"
```

The above as written will set `geom` to the first match on `relateID = joinID` rather than the Union of all matched.

**Comment by bmfekete** [comment]
You were right in both count. It took me some time to fix it, but the country import seems to work now.

**Comment by dvignoles** [comment]
Seems like it's fixed as of 5efa1249e76cf7792a540dabb4bfe596794dc494 !
